aasemble/django/apps/api/tests.py

```diff
@@ -500,6 +499,45 @@ def test_github_push_event(self, poll_one):
         github_push_event("https://github.com/eric/project0")
         poll_one.delay.assert_called_with(1)
 
+class APIv2TagsTests(APIv1Tests):
+    list_url = '/api/v2/snapshots/'
+
+    def test_create_snapshot_tag(self):
+        data = {'mirrorset': 'http://testserver/api/v2/mirror_sets/60d0ba66-d343-404b-a6e6-5c141db11a54/',
+                'tags':[{'tag':'firsttag'}, {'tag':'secondtag'}]}
```

Comment:
You're still passing `{'tags': [{'tag': 'firsttag'}, {'tag': 'secondtag'}]}`

It needs to be: `{'tags': ['firsttag', 'secondtag']}`